FAQ.md

```diff
@@ -29,6 +31,9 @@
 
 ## Builds
 
+### I do not see my repo in the list
+If in the console during app creation you see a list of repos in your org but not the one you're searching for, it may be due to having a large number of repos and the target repo not having been recently updated. If this is the case, you can make an update to your repository and your repo should appear.
```

Comment:
Small nit, feel like this can be simplified. Something like - 

> During application creation, the target repository may not show as a search result in the organization repository list if it _hasn't_ been recently updated.  
>
> This may occur if the organization has a large number of repositories.
>
> If this is the case, make an update to the repository and it should appear in the list.

Reply:
sounds good